tools/config-generator/testutils_test.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func ResetOutput() {
+	output = &bytes.Buffer{}
+}
+
+func GetOutput() string {
+	return output.(*bytes.Buffer).String()
+}
+
+func AssertOutput(t *testing.T, expected string) {
+	s := GetOutput()
+	if s != expected {
```

Comment:
The output of cmp is more user friendly, see https://chromium.googlesource.com/external/github.com/google/go-cmp/+/refs/heads/priority/cmp/example_test.go#94